### ISSUE TITLE
Getting the list of players of my ongoing match

### ISSUE THREAD

**Issue by josephmahede** [odota/web#2814]
**Is your feature request related to a problem? Please describe.**
I want to make an application where when I start a match in Dota, I get a notification from it and I can see details of my teammates and opponents. Something like Overwolf's Dotaplus, but on mobile.  A lot of people like to check their teammates and opponents' performance and best heroes while in game.

**Describe the solution you'd like**
A solution might be when I give my player ID, the API could detect that a match has started and give me the player IDs of the other 9 players in the match so that I can make additional calls on these player IDs to fetch info.

**Describe alternatives you've considered**
I don't quite know when a matchID is created, whether when a game has started or ended, so I don't quite know how to use exsiting calls to the odota API to fetch this info. The only way I found was Overwolf's match_detected api call which I assume only works in PC environments when the game is actually running?

**Comment by howardchung** [comment]
I don't think we support getting data for matches until after they're done (with the exception of the top public live games feature)
